src/components/general/Button/components/Anchor.tsx

```diff
@@ -16,30 +18,50 @@ type AnchorProps = {
 
 const AnchorComponent = forwardRef<HTMLAnchorElement, AnchorProps>(
     (
-        { children, url, targetBlank, onMouseDown, onMouseUp, onClick, onClickCapture, ...props },
+        { children, url, targetBlank, onMouseDown, onMouseUp, onClick, onClickCapture, relativeUrl, ...props },
         ref
-    ) => (
-        <a
-            className={getButtonClasses(props)}
-            href={url}
-            target={targetBlank ? '_blank' : '_self'}
-            onMouseDown={onMouseDown}
-            onMouseUp={onMouseUp}
-            onClick={onClick}
-            onClickCapture={onClickCapture}
-            ref={ref}
-        >
-            <Content>{children}</Content>
-        </a>
-    )
+    ) => {
+        if (Boolean(relativeUrl)) {

```

Comment:
Should not be necessary to parse this to boolean explicitly I think.

Reply:
Removed `Boolean`